code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -289,3 +289,12 @@
 	Neutral: Corn Oil, Ryetalyn, Tonio, Space Drugs
 	Bad: Mutagen, Polytrinic Acid, Lexorin, Zombie Powder
 	*/
+
+/obj/item/weapon/reagent_containers/pill/nanobot
+	name = "Nanobot pill"
+	desc = "Experimental medication."
+	icon_state = "pill7" //grey oblong
+
+	New()
```

Comment:
If you're adding new code, it's better to use current coding standards rather than whatever coding standards are already in the file.

It's even *better* to go through whatever file you're touching and change the code to use the current coding standards, in this case removing the partial pathing, but it's not required.